// smogon/build/latest-DHRRMYRV.mjs
// latest.json
var aaaubers = [["2016-08", 183], ["2016-07", 13477]];
var abilityunity = [["2016-05", 32], ["2016-04", 33443]];
var averagemons = [["2015-07", 49], ["2015-06", 10483]];
var battlecup1v1 = ["2015-06", 11156];
var battleofhoenn = ["2015-01", 23923];
var battlespotspecial10 = [["2015-07", 117], ["2015-06", 500]];
var battlespotspecial11 = [["2015-09", 226], ["2015-07", 2097]];
var battlespotspecial12 = [["2015-11", 128], ["2015-10", 441]];
var battlespotspecial13 = [["2016-01", 4], ["2015-11", 1020]];
var battlespotspecial17 = [["2016-10", 4], ["2016-07", 2948]];
var battlespotspecial7 = [["2015-01", 6], ["2014-12", 1058]];
var battlespotspecial8 = [["2015-02", 6], ["2015-01", 8]];
var bhdoubles = [["2016-10", 570], ["2016-09", 21177]];
var capcrucibelleplaytest = [["2016-02", 73], ["2016-01", 2036]];
var capnaviathanplaytest = [["2015-07", 97], ["2015-06", 1491]];
var capplasmantaplaytest = ["2014-11", 7];
var challengecup = [["2015-06", 4951], ["2015-05", 11331]];
var challengecup1v1 = [["2016-12", 334], ["2016-11", 129504]];
var challengecup1vs1 = [["2015-05", 2927], ["2015-03", 8136]];
var classichackmons = [["2015-03", 230], ["2015-02", 26986]];
var custompower = [["2016-07", 26], ["2016-06", 12146]];
var doublesoususpecttest = ["2016-09", 92627];
var doublesubers = ["2016-11", 27978];
var dragoncup = ["2015-02", 135];
var enchanteditems = [["2016-08", 739], ["2016-07", 39537]];
var enterthedragontype = ["2015-03", 5166];
var extremetiershift = [["2016-06", 63], ["2016-05", 52861]];
var festivefeud = [["2015-12", 3200], ["2015-11", 4470]];
var finaldestination = ["2015-04", 1545];
var flashclash = [["2016-05", 105], ["2016-04", 20215]];
var followtheleader = [["2016-09", 16], ["2016-08", 23399]];
var fu = [["2015-02", 262], ["2015-01", 33291]];
var gen11v1 = [["2023-05", 4], ["2018-03", 1695]];
var gen1challengecup = [["2018-08", 122], ["2018-07", 2047]];
var gen1customgame = ["2021-05", 1];
var gen1nintendocup1997 = ["2022-09", 2155];
var gen1nobig4ou = [["2019-07", 110], ["2017-01", 490]];
var gen1nu = ["2022-02", 1645];
var gen1ou = ["2023-10", 26202];
var gen1oubeta = ["2014-12", 488];
var gen1oumew = [["2017-08", 1], ["2017-07", 780]];
var gen1ounodefenseboosting = ["2020-02", 79];
var gen1ounodefensecurlhardenorreflect = ["2020-02", 11];
var gen1ounotauros = [["2019-02", 1], ["2019-01", 157]];
var gen1pu = [["2023-10", 23], ["2023-09", 3960]];
var gen1randombattle = ["2017-02", 79370];
var gen1stabmons = ["2018-01", 331];
var gen1stadium = ["2018-03", 3];
var gen1stadiumou = [["2023-08", 5], ["2023-07", 406]];
var gen1tradebacksou = ["2023-04", 907];
var gen1ubers = [["2023-07", 8], ["2023-06", 2888]];
var gen1uu = [["2023-01", 19], ["2022-12", 1851]];
var gen21v1 = [["2020-01", 1], ["2019-12", 111]];
var gen2inverseou = ["2019-03", 34];
var gen2mediocremons = [["2018-09", 1], ["2018-08", 126]];
var gen2nc2000crystalmoves = [["2021-11", 58], ["2021-10", 4605]];
var gen2nu = ["2023-10", 1768];
var gen2ou = ["2023-10", 10270];
var gen2oubeta = ["2014-12", 181];
var gen2oublitz = [["2019-09", 2], ["2019-08", 91]];
var gen2ouhoohcelebi = [["2018-03", 2], ["2018-02", 559]];
var gen2ouitemclause = ["2019-02", 32];
var gen2ounosnorlax = [["2017-05", 2], ["2017-04", 244]];
var gen2pu = [["2020-11", 1], ["2020-10", 1276]];
var gen2stabmons = ["2017-08", 231];
var gen2ubers = [["2023-08", 7], ["2023-07", 1240]];
var gen2uu = [["2023-02", 2], ["2023-01", 2025]];
var gen2uubl = [["2022-11", 1], ["2022-10", 1494]];
var gen2uumuk = ["2023-10", 663];
var gen31v1 = [["2022-08", 159], ["2022-07", 20497]];
var gen3adv200 = [["2019-01", 42], ["2017-02", 144]];
var gen3bl = [["2017-10", 4], ["2017-09", 81]];
var gen3customgame = ["2021-05", 4];
var gen3doublesou = [["2022-12", 7], ["2022-11", 3666]];
var gen3hoenngaiden = [["2023-10", 71], ["2023-09", 303]];
var gen3lc = [["2023-09", 4], ["2023-08", 2840]];
var gen3nu = [["2021-07", 9], ["2021-06", 2317]];
var gen3ou = ["2023-10", 283861];
var gen3oubeta = [["2015-03", 467], ["2015-01", 1791]];
var gen3oulatias = ["2019-07", 129];
var gen3ounospikes = ["2019-03", 84];
var gen3ounotyranitar = [["2018-04", 1], ["2018-03", 121]];
var gen3stabmons = ["2018-07", 237];
var gen3tiershift = [["2018-10", 3], ["2018-09", 62]];
var gen3ubers = [["2023-09", 3], ["2023-08", 7950]];
var gen3ubersbeta = [["2015-03", 4], ["2014-12", 25]];
var gen3uu = [["2023-03", 24], ["2023-02", 1927]];
var gen3uubl = [["2020-05", 4], ["2020-04", 387]];
var gen41v1 = [["2020-03", 1], ["2020-02", 745]];
var gen4anythinggoes = [["2022-06", 91], ["2022-05", 5664]];
var gen4balancedhackmons = ["2017-12", 90];
var gen4customgame = ["2021-05", 3];
var gen4doublesou = [["2023-09", 123], ["2023-08", 620]];
var gen4lc = ["2022-09", 2544];
var gen4monotype = ["2020-12", 1903];
var gen4nu = [["2023-03", 12], ["2023-02", 1541]];
var gen4ou = ["2023-10", 77580];
var gen4oulatimence = ["2017-05", 148];
var gen4outeampreview = [["2018-07", 2], ["2018-06", 164]];
var gen4pu = [["2018-12", 62], ["2018-04", 65]];
var gen4ubers = [["2023-10", 84], ["2023-09", 26760]];
var gen4uu = [["2023-04", 22], ["2023-03", 5071]];
var gen4zu = [["2023-02", 34], ["2023-01", 2330]];
var gen51v1 = ["2023-09", 3915];
var gen52v2doubles = ["2020-09", 1009];
var gen5anythinggoes = [["2017-01", 64], ["2016-12", 8640]];
var gen5balancedhackmons = ["2019-05", 156];
var gen5customgame = ["2018-03", 3];
var gen5doublescustomgame = ["2018-03", 1];
var gen5doublesou = [["2022-01", 15], ["2021-12", 1582]];
var gen5gbudoubles = [["2018-01", 1], ["2015-05", 2]];
var gen5gbusingles = [["2015-06", 9], ["2015-04", 4375]];
var gen5lc = [["2022-02", 6], ["2022-01", 370]];
var gen5monotype = ["2018-10", 525];
var gen5nu = [["2022-05", 9], ["2022-04", 1183]];
var gen5ou = ["2023-10", 59781];
var gen5oublind = ["2017-06", 171];
var gen5pu = [["2022-04", 12], ["2022-03", 1751]];
var gen5ru = [["2023-03", 10], ["2023-02", 1730]];
var gen5smogondoubles = [["2015-04", 1], ["2015-03", 25]];
var gen5ubers = ["2023-10", 12966];
var gen5uu = ["2023-04", 4862];
var gen5vgc2013 = ["2023-05", 3];
var gen5zu = [["2023-04", 2], ["2023-03", 1631]];
var gen61v1 = [["2022-01", 14], ["2016-11", 243201]];
var gen6almostanyability = [["2021-05", 131], ["2016-10", 7600]];
var gen6anythinggoes = [["2021-06", 5], ["2017-07", 13973]];
var gen6balancedhackmons = [["2019-07", 2], ["2016-11", 90551]];
var gen6battlefactory = [["2018-12", 789], ["2018-02", 13353]];
var gen6battlespotdoubles = [["2018-01", 1], ["2014-12", 112633]];
var gen6battlespotsingles = [["2021-06", 3], ["2016-11", 112405]];
var gen6battlespottriples = [["2017-12", 7], ["2016-12", 3038]];
var gen6cap = [["2021-05", 4], ["2016-11", 10305]];
var gen6customgame = ["2021-05", 8];
var gen6doublesou = [["2023-10", 2652], ["2017-01", 24352]];
var gen6doublesuu = [["2017-07", 3], ["2016-11", 14863]];
var gen6lc = [["2023-02", 13], ["2016-11", 68009]];
var gen6mixandmega = [["2018-09", 5], ["2016-11", 46660]];
var gen6monotype = [["2020-11", 1], ["2016-11", 232353]];
var gen6nu = [["2020-01", 2], ["2016-12", 13243]];
var gen6ou = ["2023-10", 67178];
var gen6ounoteampreview = [["2018-01", 8], ["2017-12", 520]];
var gen6pu = [["2022-05", 10], ["2016-11", 82966]];
var gen6purehackmons = ["2023-10", 26391];
var gen6randombattle = ["2017-02", 20405];
var gen6ru = [["2023-04", 1839], ["2016-11", 71167]];
var gen6triplescustomgame = ["2018-01", 1];
var gen6ubers = [["2023-05", 1], ["2022-11", 14204]];
var gen6uu = [["2023-06", 17], ["2017-01", 27784]];
var gen6vgc2016 = [["2021-05", 1], ["2016-11", 160180]];
var gen71v1 = [["2021-05", 1], ["2020-11", 12939]];
var gen71v1suspecttest = ["2018-02", 10856];
var gen72017noholdsbarred = ["2017-08", 26487];
var gen72v2doubles = [["2018-11", 30], ["2018-10", 14847]];
var gen7350cup = [["2019-11", 80], ["2019-10", 20711]];
var gen7almostanyability = [["2019-11", 4116], ["2019-03", 9462]];
var gen7almostanyabilitysuspecttest = [["2018-01", 707], ["2017-12", 5835]];
var gen7alolafriendly = [["2017-02", 6], ["2017-01", 31637]];
var gen7alphabetcup = [["2018-02", 468], ["2018-01", 22794]];
var gen7anythinggoes = ["2022-11", 84902];
var gen7anythinggoesbeta = ["2016-11", 53352];
var gen7averagemons = [["2018-12", 35], ["2018-11", 18468]];
var gen7balancedhackmons = [["2022-12", 622], ["2022-03", 16929]];
var gen7balancedhackmonssuspecttest = [["2018-08", 8894], ["2018-07", 12031]];
var gen7battlefactory = ["2018-12", 96762];
var gen7battleofalola = ["2016-12", 35226];
var gen7battlespotdoubles = [["2019-11", 3788], ["2018-06", 10438]];
var gen7battlespotsingles = ["2019-11", 42585];
var gen7battlespotspecial1 = [["2017-01", 21], ["2016-12", 301]];
var gen7battlespotspecial10 = [["2018-07", 1], ["2018-05", 462]];
var gen7battlespotspecial11 = [["2018-08", 237], ["2018-07", 721]];
var gen7battlespotspecial12 = [["2018-11", 1], ["2018-09", 255]];
var gen7battlespotspecial13 = [["2018-12", 5], ["2018-11", 629]];
var gen7battlespotspecial14 = [["2019-03", 4], ["2019-01", 25]];
var gen7battlespotspecial15 = [["2019-06", 12], ["2019-04", 3064]];
var gen7battlespotspecial16 = [["2019-08", 7], ["2019-06", 2606]];
var gen7battlespotspecial17 = [["2019-11", 1], ["2019-09", 2326]];
var gen7battlespotspecial2 = [["2017-03", 616], ["2017-02", 1256]];
var gen7battlespotspecial3 = [["2017-05", 24], ["2017-03", 144]];
var gen7battlespotspecial4 = [["2017-07", 10], ["2017-05", 750]];
var gen7battlespotspecial5 = [["2017-09", 250], ["2017-07", 1541]];
var gen7battlespotspecial6 = [["2017-11", 63], ["2017-09", 942]];
var gen7battlespotspecial7 = [["2018-01", 22], ["2017-12", 47]];
var gen7battlespotspecial8 = [["2018-03", 79], ["2018-01", 1448]];
var gen7battlespotspecial9 = [["2018-04", 4], ["2018-03", 371]];
var gen7benjaminbutterfree = ["2018-09", 16];
var gen7bssfactory = ["2018-12", 18935];
var gen7camomons = [["2019-11", 1109], ["2017-04", 59625]];
var gen7cap = [["2020-01", 84], ["2019-06", 10904]];
var gen7caplc = ["2018-03", 1];
var gen7challengecup = ["2016-11", 26024];
var gen7challengecup1v1 = ["2018-12", 98097];
var gen7challengecup2v2 = ["2018-03", 12];
var gen7challengecupbeta = ["2016-11", 10628];
var gen7chimera = [["2019-04", 40], ["2019-03", 27416]];
var gen7crossevolution = [["2019-09", 73], ["2019-08", 18888]];
var gen7customgame = [["2021-05", 5], ["2018-03", 43]];
var gen7doublesanythinggoesbeta = ["2016-11", 6888];
var gen7doublescustomgame = [["2018-04", 1], ["2018-03", 30]];
var gen7doubleshackmonscup = [["2018-04", 1], ["2018-03", 49]];
var gen7doublesou = ["2023-10", 13536];
var gen7doublesoubeta = ["2016-11", 38];
var gen7doublesoususpecttest = ["2018-05", 54625];
var gen7doublesubers = ["2019-11", 17263];
var gen7doublesuu = [["2019-11", 9739], ["2019-10", 16010]];
var gen7dragoncup = [["2018-01", 1295], ["2017-11", 12619]];
var gen7dualwielding = [["2017-12", 93], ["2017-11", 18529]];
var gen7followtheleader = [["2019-08", 39], ["2019-07", 31354]];
var gen7fortemons = [["2018-12", 71], ["2018-11", 54918]];
var gen7fullpotential = [["2017-12", 338], ["2017-11", 25858]];
var gen7godlygift = [["2019-05", 238], ["2019-04", 65358]];
var gen7godsandfollowers = [["2018-11", 61], ["2018-10", 41524]];
var gen7hackmonscup = ["2018-12", 48123];
var gen7hiddentype = [["2019-03", 3], ["2019-02", 8655]];
var gen7inheritance = [["2019-11", 168], ["2019-10", 27376]];
var gen7inverse = [["2018-06", 89], ["2018-05", 28228]];
var gen7johtoxalola = ["2017-12", 10867];
var gen7kantoxalolaregionalrumble = ["2017-03", 42001];
var gen7lastwill = [["2017-06", 83], ["2017-05", 23009]];
var gen7lc = [["2022-04", 10], ["2019-11", 26585]];
var gen7lcsuspecttest = ["2018-03", 46340];
var gen7letsgo1v1 = ["2019-01", 2298];
var gen7letsgocup = [["2018-08", 7289], ["2018-07", 59877]];
var gen7letsgodoublesnorestrictions = [["2019-02", 15], ["2018-12", 166]];
var gen7letsgodoublesou = [["2019-11", 26], ["2019-01", 490]];
var gen7letsgohackmons = [["2019-01", 3], ["2018-12", 755]];
var gen7letsgoou = [["2023-09", 2], ["2019-04", 10883]];
var gen7letsgosinglesnorestrictions = [["2019-02", 374], ["2019-01", 1029]];
var gen7letsgouu = [["2022-09", 11], ["2022-08", 396]];
var gen7linked = [["2019-09", 522], ["2019-08", 64187]];
var gen7mediocremons = [["2018-01", 89], ["2017-12", 12377]];
var gen7megamelee = ["2017-09", 22039];
var gen7megamons = [["2019-10", 59], ["2019-09", 24429]];
var gen7mergemons = [["2019-10", 10], ["2019-09", 26739]];
var gen7metagamiate = [["2018-08", 371], ["2018-07", 26395]];
var gen7metronomebattle = ["2018-04", 112062];
var gen7middlecup = [["2018-03", 6], ["2018-02", 9750]];
var gen7mixandmega = ["2023-03", 14775];
var gen7mixandmegalc = [["2019-05", 41], ["2019-04", 13503]];
var gen7mixandmegasuspecttest = ["2018-04", 20344];
var gen7monotype = [["2023-06", 17], ["2020-10", 35548]];
var gen7monotypesuspecttest = ["2017-12", 162577];
var gen7natureswap = [["2019-08", 77], ["2019-07", 18645]];
var gen7nfe = [["2019-07", 90], ["2019-06", 19251]];
var gen7nu = [["2023-06", 10], ["2019-11", 43826]];
var gen7nualpha = ["2017-06", 44168];
var gen7nubeta = ["2017-08", 33407];
var gen7nususpecttest = [["2018-07", 5073], ["2018-06", 37722]];
var gen7ou = ["2023-10", 159606];
var gen7oubeta = ["2016-11", 1441];
var gen7oublitz = ["2019-11", 91483];
var gen7ouprebank = ["2016-11", 3638];
var gen7oususpecttest = ["2017-09", 1026428];
var gen7partnersincrime = [["2019-10", 35], ["2018-09", 48860]];
var gen7petmodcleanslatemicro = [["2020-03", 2], ["2020-01", 191]];
var gen7pokbilities = [["2017-02", 65], ["2017-01", 40982]];
var gen7pokebankag = ["2016-11", 1434];
var gen7pokebankanythinggoes = ["2017-01", 532909];
var gen7pokebankanythinggoesbeta = ["2016-11", 8];
var gen7pokebankdoublesag = ["2016-11", 109];
var gen7pokebankdoublesou = ["2017-01", 234237];
var gen7pokebankdoublesoubeta = ["2016-11", 2];
var gen7pokebankdoublesubers = ["2017-01", 34528];
var gen7pokebanklc = ["2017-01", 54470];
var gen7pokebankou = ["2017-01", 1141698];
var gen7pokebankoubeta = ["2016-11", 16];
var gen7pokebankoususpecttest = ["2017-01", 344597];
var gen7pokebankubers = ["2017-01", 253817];
var gen7pokebankuualpha = ["2016-12", 98346];
var gen7pokebilities = [["2019-03", 78], ["2019-02", 54082]];
var gen7prebankou = ["2017-01", 836064];
var gen7pu = [["2022-11", 44], ["2019-11", 57390]];
var gen7pualpha = ["2017-08", 14072];
var gen7pubeta = ["2017-09", 25158];
var gen7purehackmons = [["2023-10", 117], ["2023-09", 36351]];
var gen7pususpecttest = ["2018-05", 29442];
var gen7randombattle = ["2017-02", 4304257];
var gen7randomdoublesbattle = ["2017-02", 369322];
var gen7reversed = [["2018-08", 315], ["2018-07", 17491]];
var gen7ru = [["2022-06", 19], ["2019-12", 10570]];
var gen7rualpha = ["2017-03", 41092];
var gen7rubeta = ["2017-06", 29301];
var gen7rususpecttest = [["2018-08", 8930], ["2018-07", 48083]];
var gen7scalemons = [["2018-07", 83], ["2018-06", 30074]];
var gen7sharedpower = [["2018-03", 518], ["2018-02", 209191]];
var gen7simplesymphony = [["2019-02", 19], ["2019-01", 15504]];
var gen7sketchmons = [["2019-02", 52], ["2019-01", 33944]];
var gen7sketchmonssuspecttest = [["2017-11", 96], ["2017-01", 1721]];
var gen7stabmons = [["2021-06", 3], ["2018-02", 10957]];
var gen7stabmonssuspecttest = [["2018-05", 77], ["2018-04", 3857]];
var gen7suicidecup = [["2018-07", 89], ["2018-06", 29454]];
var gen7tiershift = [["2019-12", 214], ["2019-11", 42284]];
var gen7tinytourney = ["2017-06", 36500];
var gen7trademarked = [["2019-06", 223], ["2019-05", 58964]];
var gen7ubers = [["2023-01", 762], ["2022-12", 42469]];
var gen7uberssuspecttest = ["2018-07", 76507];
var gen7ultimatez = [["2019-06", 111], ["2019-05", 24282]];
var gen7ultrafinal = [["2019-11", 1343], ["2019-09", 8928]];
var gen7ultrakalosclassic = [["2018-05", 3], ["2018-04", 7677]];
var gen7ultranoholdsbarred = [["2018-07", 31], ["2018-05", 8639]];
var gen7ultrasinnohclassic = [["2018-03", 94], ["2018-02", 23083]];
var gen7ultraspookycup = [["2018-11", 12], ["2018-10", 27522]];
var gen7uu = [["2023-07", 16], ["2020-10", 12027]];
var gen7uubeta = [["2017-02", 1126], ["2017-01", 173786]];
var gen7uususpecttest = ["2018-06", 109713];
var gen7vgc2017 = [["2021-06", 2], ["2017-12", 22683]];
var gen7vgc2017beta = ["2016-11", 8145];
var gen7vgc2018 = [["2019-11", 582], ["2018-11", 16190]];
var gen7vgc2019moonseries = [["2019-04", 1945], ["2019-03", 148609]];
var gen7vgc2019sunseries = ["2018-12", 114025];
var gen7vgc2019ultraseries = [["2020-02", 52], ["2019-11", 119062]];
var gen7weaknesscup = ["2017-07", 31581];
var gen7zu = [["2020-02", 7], ["2019-11", 15376]];
var gen81v1 = ["2022-11", 138736];
var gen82v2doubles = [["2022-11", 4128], ["2022-10", 7480]];
var gen8350cup = [["2021-11", 44], ["2021-10", 15703]];
var gen8aaadoubles = [["2021-12", 57], ["2021-11", 14145]];
var gen8almostanyability = ["2022-11", 20973];
var gen8alphabetcup = [["2021-09", 85], ["2021-08", 28884]];
var gen8alternatium = [["2022-02", 2], ["2022-01", 3]];
var gen8anythinggoes = [["2023-05", 1], ["2023-02", 87967]];
var gen8anythinggoesobtainable = ["2021-05", 1];
var gen8anythinggoeswitheternamax = ["2021-04", 3420];
var gen8armorbeginnings = ["2020-07", 9010];
var gen8balancedhackmons = ["2022-11", 13964];
var gen8balancedhackmonsbeta = ["2019-11", 7345];
var gen8battleoflegends = ["2020-10", 55006];
var gen8battlestadiumdoubles = ["2020-10", 209968];
var gen8battlestadiumdoublesseries13 = [["2022-11", 1254], ["2022-10", 105417]];
var gen8battlestadiumsingles = [["2023-01", 8], ["2022-11", 15082]];
var gen8bdsp3v3singles = [["2022-03", 618], ["2022-01", 5910]];
var gen8bdspbalancedhackmons = [["2022-01", 74], ["2021-12", 10236]];
var gen8bdspbattlefestivaldoubles = [["2022-08", 65], ["2022-01", 48831]];
var gen8bdspdoublesou = [["2022-12", 18], ["2022-03", 12348]];
var gen8bdsplc = [["2022-08", 3], ["2022-01", 4142]];
var gen8bdspmonotype = [["2022-08", 51], ["2022-01", 16705]];
var gen8bdspnu = [["2022-08", 8], ["2022-03", 20363]];
var gen8bdspou = [["2023-07", 6], ["2022-11", 18931]];
var gen8bdsppu = [["2022-08", 32], ["2022-05", 4286]];
var gen8bdspru = [["2022-08", 17], ["2022-02", 24002]];
var gen8bdspubers = [["2022-08", 128], ["2021-12", 19309]];
var gen8bdspuu = [["2022-08", 115], ["2022-02", 24150]];
var gen8bonustype = [["2021-07", 275], ["2021-06", 42907]];
var gen8brokenrecord = [["2022-08", 51], ["2022-07", 14071]];
var gen8camomons = [["2022-02", 107], ["2022-01", 15902]];
var gen8cap = [["2022-12", 90], ["2022-10", 20461]];
var gen8cap1v1 = [["2021-09", 213], ["2020-05", 4061]];
var gen8categoryswap = [["2022-09", 23], ["2022-08", 19456]];
var gen8championleague = [["2022-10", 185], ["2022-09", 8923]];
var gen8chimera1v1 = [["2022-09", 5319], ["2021-06", 17240]];
var gen8crossevolution = [["2022-04", 141], ["2022-03", 22456]];
var gen8doublescustomgame = ["2021-05", 9];
var gen8doublesou = ["2023-10", 11130];
var gen8doublesoubeta = ["2019-11", 19726];
var gen8doublesoudynamaxlevel0 = ["2020-07", 17667];
var gen8doublesubers = [["2022-12", 8080], ["2022-11", 46671]];
var gen8doublesuu = [["2022-12", 495], ["2022-11", 13374]];
var gen8dragonkingcup = ["2020-12", 22379];
var gen8extremespeedsinglebattles = ["2020-03", 24701];
var gen8fairandsquare = [["2021-07", 11], ["2021-06", 4611]];
var gen8firstblood = [["2021-08", 5], ["2021-07", 32449]];
var gen8flipped = [["2022-05", 91], ["2022-04", 15449]];
var gen8fu = ["2022-04", 293];
var gen8fusionevolutionuu = [["2021-10", 1], ["2021-07", 7]];
var gen8galarbeginnings = ["2019-12", 46135];
var gen8galarnewcomers = [["2020-03", 169], ["2020-01", 25516]];
var gen8godlygift = [["2022-11", 1511], ["2022-07", 10857]];
var gen8gymchallenge = ["2021-11", 8733];
var gen8ichoosechu = [["2022-09", 273], ["2022-08", 8712]];
var gen8inheritance = [["2022-08", 122], ["2022-07", 22004]];
var gen8inverse = [["2022-07", 26], ["2022-06", 8831]];
var gen8jumpmagikarp = ["2022-07", 29426];
var gen8lc = [["2023-03", 2], ["2022-11", 28832]];
var gen8lcsuspecttest = ["2020-08", 315];
var gen8linked = [["2022-03", 38], ["2022-02", 37126]];
var gen8maxberries = [["2021-04", 4], ["2021-03", 25983]];
var gen8megamax = [["2020-10", 10], ["2020-07", 660]];
var gen8megasforall = [["2021-04", 3364], ["2021-03", 9554]];
var gen8metronomebattle = ["2022-11", 39935];
var gen8mixandmega = [["2022-11", 5763], ["2022-10", 11462]];
var gen8monotype = [["2023-03", 112], ["2022-12", 15099]];
var gen8monotypedynamax = ["2021-04", 1513];
var gen8multibility = [["2021-11", 358], ["2021-10", 86367]];
var gen8mythicalmelee = ["2021-09", 4878];
var gen8natdexagwitheternamax = ["2021-04", 2210];
var gen8natdexanythinggoes = ["2019-11", 12470];
var gen8nationaldex = ["2023-10", 14800];
var gen8nationaldexag = ["2022-11", 403232];
var gen8nationaldexbeta = ["2019-12", 110443];
var gen8nationaldexbh = [["2022-12", 231], ["2020-11", 13030]];
var gen8nationaldexmonotype = ["2022-11", 62290];
var gen8nationaldexnostall = ["2021-04", 2394];
var gen8nationaldexuu = [["2022-12", 487], ["2022-11", 21176]];
var gen8nationaldexuunostall = ["2021-04", 32];
var gen8natureswap = [["2021-03", 189], ["2021-02", 46352]];
var gen8ndmonotypenotoxapex = ["2021-04", 83];
var gen8nfe = [["2022-11", 134], ["2021-05", 3261]];
var gen8nu = [["2023-03", 4], ["2022-11", 23264]];
var gen8nususpecttest = ["2021-04", 58];
var gen8ou = ["2023-10", 71858];
var gen8oubeta = ["2019-11", 298355];
var gen8oublitz = ["2022-11", 12248];
var gen8oususpecttest = ["2021-04", 6409];
var gen8pacifistmons = [["2020-05", 6], ["2020-04", 4803]];
var gen8partnersincrime = [["2022-10", 5], ["2022-09", 38210]];
var gen8pokebilities = [["2022-02", 153], ["2022-01", 20921]];
var gen8pokebilitiesaaa = [["2022-06", 1094], ["2022-05", 77591]];
var gen8pu = [["2023-03", 4], ["2022-11", 36640]];
var gen8puindeedeefnoteampreview = ["2021-04", 582];
var gen8purehackmons = [["2022-12", 71], ["2022-11", 67e3]];
var gen8reevolution = [["2022-09", 318], ["2022-08", 20695]];
var gen8revelationmons = [["2022-11", 254], ["2022-10", 32101]];
var gen8ru = [["2023-03", 2], ["2022-11", 32191]];
var gen8rudynamax = ["2021-04", 889];
var gen8samedoublebeat = ["2021-08", 1513];
var gen8scalemons = [["2020-12", 11], ["2020-03", 19031]];
var gen8sharedpower = [["2022-04", 664], ["2022-03", 60729]];
var gen8singlebattlecup = [["2022-07", 359], ["2022-06", 5503]];
var gen8sketchmons = [["2022-05", 291], ["2022-04", 28146]];
var gen8spikemuthcup = [["2023-01", 1], ["2022-11", 32237]];
var gen8spookycup2021 = [["2021-11", 21], ["2021-10", 30088]];
var gen8staaabmons = [["2021-01", 101], ["2020-12", 68776]];
var gen8stabmons = [["2022-11", 1225], ["2022-06", 4579]];
var gen8stabmonsmixandmega = [["2021-09", 214], ["2021-08", 35555]];
var gen8tagteamsingles = [["2022-11", 10], ["2022-10", 6431]];
var gen8testyourluck = [["2022-01", 5839], ["2021-12", 11063]];
var gen8thelosersgame = [["2021-04", 12], ["2021-03", 22674]];
var gen8tiershift = [["2022-07", 171], ["2022-06", 41038]];
var gen8tiershiftcamomonsobtainable = ["2021-06", 1];
var gen8tinytourney = ["2021-12", 6791];
var gen8trademarked = [["2022-03", 69], ["2022-02", 22103]];
var gen8tundratourney = ["2020-11", 11496];
var gen8ubers = [["2023-05", 1], ["2023-02", 14403]];
var gen8ultimatefinale = [["2022-11", 32], ["2022-10", 3645]];
var gen8uu = [["2023-08", 1], ["2022-11", 71767]];
var gen8uususpecttest = ["2021-04", 332];
var gen8vgc2020 = [["2023-05", 1], ["2020-10", 417645]];
var gen8vgc2021 = [["2021-05", 3740], ["2021-04", 696293]];
var gen8vgc2021series10 = [["2021-11", 953], ["2021-10", 489659]];
var gen8vgc2021series11 = [["2022-02", 1018], ["2022-01", 136323]];
var gen8vgc2021series9 = [["2021-08", 530], ["2021-07", 211948]];
var gen8vgc2021series9limitonerestrictedrestrictedlegendary = [
  ["2021-06", 2],
  ["2021-05", 3]
];
var gen8vgc2022 = [["2023-01", 1827], ["2022-12", 23336]];
var gen8viabilities = [["2021-01", 5], ["2020-10", 248]];
var gen8waterparadise = ["2021-07", 23412];
var gen8zu = [["2023-04", 2], ["2022-10", 17108]];
var gen91v1 = ["2023-10", 115048];
var gen92v2doubles = [["2023-10", 3556], ["2023-01", 11873]];
var gen9350cup = [["2023-09", 4], ["2023-08", 29218]];
var gen9almostanyability = ["2023-10", 49684];
var gen9alphabetcup = [["2023-07", 310], ["2023-06", 39729]];
var gen9anythinggoes = ["2023-10", 308983];
var gen9balancedhackmons = ["2023-10", 39391];
var gen9battlestadiumdoubles = [["2022-12", 4151], ["2022-11", 413572]];
var gen9battlestadiumsingles = [["2022-12", 1165], ["2022-11", 142491]];
var gen9battlestadiumsinglesregulationc = [["2023-07", 246], ["2023-06", 42472]];
var gen9battlestadiumsinglesregulationd = [["2023-10", 279], ["2023-09", 59482]];
var gen9battlestadiumsinglesregulatione = ["2023-10", 132149];
var gen9battlestadiumsinglesseries1 = [["2023-02", 2413], ["2023-01", 256579]];
var gen9battlestadiumsinglesseries2 = [["2023-04", 4947], ["2023-03", 103080]];
var gen9bonustype = [["2023-09", 21], ["2023-08", 29635]];
var gen9brokencup = [["2023-05", 34], ["2023-04", 145541]];
var gen9camomons = [["2023-03", 18], ["2023-02", 8064]];
var gen9cap = [["2023-10", 2571], ["2023-08", 8558]];
var gen9convergence = ["2023-10", 40245];
var gen9crossevolution = [["2023-05", 1], ["2023-04", 34851]];
var gen9doubleslc = [["2023-09", 52], ["2023-02", 1094]];
var gen9doublesou = ["2023-10", 293858];
var gen9doublesubers = ["2023-10", 50472];
var gen9doublesuu = ["2023-10", 22133];
var gen9forceofthefallen = [["2023-08", 23], ["2023-07", 36731]];
var gen9fortemons = [["2023-05", 1], ["2023-03", 146507]];
var gen9franticfusions = [["2023-10", 74], ["2023-09", 68601]];
var gen9freedomcup = [["2023-07", 2677], ["2023-06", 45142]];
var gen9fullpotential = [["2023-01", 37], ["2022-12", 10022]];
var gen9godlygift = ["2023-10", 26599];
var gen9inheritance = [["2023-10", 826], ["2023-07", 57832]];
var gen9kitakamiprologue = ["2023-10", 50610];
var gen9lc = ["2023-10", 34951];
var gen9meowcup = ["2023-04", 2941];
var gen9metronomebattle = ["2023-10", 58334];
var gen9mixandmega = ["2023-10", 5374];
var gen9mixandmegacup = [["2023-07", 48], ["2023-06", 12160]];
var gen9monotype = ["2023-10", 185953];
var gen9nationaldex = ["2023-10", 788332];
var gen9nationaldexag = [["2023-05", 3], ["2023-02", 12946]];
var gen9nationaldexbh = [["2023-08", 41], ["2023-07", 14853]];
var gen9nationaldexdoubles = ["2023-10", 54816];
var gen9nationaldexmonotype = ["2023-10", 85744];
var gen9nationaldexubers = ["2023-10", 264450];
var gen9nationaldexuu = ["2023-10", 27257];
var gen9nfe = [["2023-09", 27], ["2023-01", 1344]];
var gen9nu = ["2023-10", 53886];
var gen9ou = ["2023-10", 1709112];
var gen9oususpect = ["2023-04", 9904];
var gen9paldeaprologue = ["2023-03", 70251];
var gen9partnersincrime = [["2023-10", 1366], ["2023-01", 16457]];
var gen9partnersincrimesuspect = ["2023-04", 99];
var gen9pokebilities = [["2023-05", 2], ["2023-04", 72556]];
var gen9pu = ["2023-10", 65196];
var gen9ru = ["2023-10", 67803];
var gen9sharedpower = [["2023-01", 200], ["2022-12", 35293]];
var gen9sharingiscaring = ["2023-10", 7469];
var gen9stabmons = [["2023-10", 8963], ["2023-09", 12909]];
var gen9teradonation = [["2023-04", 3], ["2023-03", 3410]];
var gen9thecardgame = [["2023-06", 1], ["2023-05", 3163]];
var gen9tiershift = [["2023-06", 86], ["2023-05", 64960]];
var gen9trademarked = [["2023-07", 64], ["2023-06", 33897]];
var gen9trickmagic = ["2023-10", 13133];
var gen9ubers = ["2023-10", 131539];
var gen9uu = ["2023-10", 314568];
var gen9vgc2023regulationc = [["2023-07", 1205], ["2023-06", 285451]];
var gen9vgc2023regulationd = [["2023-10", 122], ["2023-09", 554050]];
var gen9vgc2023regulatione = ["2023-10", 1149474];
var gen9vgc2023regulationebo3 = ["2023-10", 285793];
var gen9vgc2023series1 = [["2023-02", 3899], ["2023-01", 1384164]];
var gen9vgc2023series2 = ["2023-04", 21261];
var gen9voltturnmayhem = [["2023-02", 59], ["2023-01", 12786]];
var gen9zu = [["2023-10", 8304], ["2023-05", 15242]];
var generationshowdown = ["2015-02", 4140];
var giftsofthegods = ["2016-10", 74763];
var godsandfollowers = [["2016-02", 107], ["2016-01", 83565]];
var gottalent = [["2016-07", 34], ["2016-06", 13217]];
var hackmons1v1 = [["2015-10", 690], ["2015-09", 85409]];
var hackmonscup = [["2016-12", 81], ["2016-11", 55810]];
var hiddentype = ["2015-03", 4838];
var higheststatmeta = [["2015-11", 5], ["2015-10", 3863]];
var inheritance = [["2015-12", 51], ["2015-11", 21753]];
var inversebattle = ["2016-11", 1236];
var johtoclassic = [["2016-06", 39], ["2016-05", 103707]];
var kantoclassic = ["2016-03", 25279];
var lcextended = ["2015-12", 1974];
var lcsuspecttest = ["2016-07", 38265];
var lcuu = ["2016-11", 168];
var linked = [["2015-07", 57], ["2015-06", 18397]];
var megamons = [["2016-05", 191], ["2016-04", 51161]];
var metagamiate = [["2016-04", 11], ["2016-03", 24955]];
var monogen = [["2016-02", 53], ["2016-01", 27549]];
var monsjustmons = [["2015-09", 9], ["2015-08", 3273]];
var mrboneswildride = ["2015-04", 486];
var natureswap = [["2016-09", 24], ["2016-08", 33033]];
var noguardgalaxy = [["2015-11", 110], ["2015-10", 12373]];
var nostatus = [["2015-09", 11], ["2015-08", 3239]];
var nususpecttest = ["2016-07", 89732];
var ounomega = [["2016-05", 347], ["2015-11", 56505]];
var oususpecttest = ["2016-11", 103986];
var pgllittlecup = ["2015-05", 21477];
var pikachucup = ["2015-08", 28155];
var primalbattle = ["2015-09", 6501];
var proteanpalace = [["2015-08", 54], ["2015-07", 8121]];
var pususpecttest = ["2016-12", 10318];
var randomdoublesbattle = ["2016-11", 402503];
var randomtriplesbattle = ["2016-11", 93697];
var returnd = ["2016-10", 23099];
var rususpecttest = ["2016-10", 37507];
var sametypestealthrock = ["2015-11", 1225];
var scrappyskirmish = ["2015-11", 7792];
var seasonaldimensiondoom = [["2016-04", 98], ["2016-03", 36566]];
var seasonalhanvshun = [["2015-03", 1403], ["2015-02", 8991]];
var seasonalpolaropposites = [["2016-03", 43], ["2016-01", 25843]];
var seasonalrainbowroad = [["2015-10", 69], ["2015-08", 12711]];
var seasonalsleighshowdown = [["2015-01", 501], ["2014-12", 8292]];
var seasonalspacetimefuntimes = [["2015-02", 1597], ["2015-01", 18346]];
var seasonalspoopyparty = [["2015-11", 489], ["2015-10", 14057]];
var seasonalstrikesback = [["2014-12", 1264], ["2014-11", 2687]];
var seasonalsupersquadsmackdown = [["2015-12", 80], ["2015-11", 25015]];
var seasonalsuperstaffbros = [["2015-05", 2380], ["2015-04", 28501]];
var seasonalyouarenotprepared = [["2015-07", 35], ["2015-05", 4249]];
var sinnohclassic = ["2016-07", 49283];
var sketchmons = [["2016-12", 34], ["2015-09", 37934]];
var smogondoubles = ["2015-03", 168942];
var smogondoublessuspecttest = [["2015-04", 82], ["2014-12", 34536]];
var smogondoublesubers = ["2015-03", 10558];
var smogondoublesuu = [["2015-03", 1574], ["2015-02", 2123]];
var smogontriples = [["2016-11", 9449], ["2016-10", 19464]];
var spookycup = ["2015-10", 30194];
var stabmons = [["2016-12", 33], ["2016-11", 28234]];
var statswitch = [["2015-01", 37], ["2014-12", 15796]];
var tiershift = [["2016-11", 644], ["2016-09", 1888]];
var typereflector = [["2016-06", 14], ["2016-05", 29397]];
var uberssuspecttest = ["2015-04", 184];
var unovaclassic = ["2016-08", 12416];
var unreleasedou = [["2016-04", 1], ["2016-03", 3119]];
var uunoscald = ["2015-03", 9705];
var uususpecttest = ["2016-10", 28683];
var vgc2014 = ["2014-12", 49294];
var vgc2015 = [["2017-06", 33], ["2016-11", 54650]];
var voltturnmayhem = [["2016-03", 50], ["2016-02", 18591]];
var latest_default = {
  "2v2doubles": [["2015-06", 46], ["2015-05", 2225]],
  "350cup": [["2016-01", 123], ["2015-12", 32786]],
  aaaubers,
  abilityunity,
  averagemons,
  battlecup1v1,
  battleofhoenn,
  battlespotspecial10,
  battlespotspecial11,
  battlespotspecial12,
  battlespotspecial13,
  battlespotspecial17,
  battlespotspecial7,
  battlespotspecial8,
  bhdoubles,
  capcrucibelleplaytest,
  capnaviathanplaytest,
  capplasmantaplaytest,
  challengecup,
  challengecup1v1,
  challengecup1vs1,
  classichackmons,
  custompower,
  doublesoususpecttest,
  doublesubers,
  dragoncup,
  enchanteditems,
  enterthedragontype,
  extremetiershift,
  festivefeud,
  finaldestination,
  flashclash,
  followtheleader,
  fu,
  gen11v1,
  gen1challengecup,
  gen1customgame,
  gen1nintendocup1997,
  gen1nobig4ou,
  gen1nu,
  gen1ou,
  gen1oubeta,
  gen1oumew,
  gen1ounodefenseboosting,
  gen1ounodefensecurlhardenorreflect,
  gen1ounotauros,
  gen1pu,
  gen1randombattle,
  gen1stabmons,
  gen1stadium,
  gen1stadiumou,
  gen1tradebacksou,
  gen1ubers,
  gen1uu,
  gen21v1,
  gen2inverseou,
  gen2mediocremons,
  gen2nc2000crystalmoves,
  gen2nu,
  gen2ou,
  gen2oubeta,
  gen2oublitz,
  gen2ouhoohcelebi,
  gen2ouitemclause,
  gen2ounosnorlax,
  gen2pu,
  gen2stabmons,
  gen2ubers,
  gen2uu,
  gen2uubl,
  gen2uumuk,
  gen31v1,
  gen3adv200,
  gen3bl,
  gen3customgame,
  gen3doublesou,
  gen3hoenngaiden,
  gen3lc,
  gen3nu,
  gen3ou,
  gen3oubeta,
  gen3oulatias,
  gen3ounospikes,
  gen3ounotyranitar,
  gen3stabmons,
  gen3tiershift,
  gen3ubers,
  gen3ubersbeta,
  gen3uu,
  gen3uubl,
  gen41v1,
  gen4anythinggoes,
  gen4balancedhackmons,
  gen4customgame,
  gen4doublesou,
  gen4lc,
  gen4monotype,
  gen4nu,
  gen4ou,
  gen4oulatimence,
  gen4outeampreview,
  gen4pu,
  gen4ubers,
  gen4uu,
  gen4zu,
  gen51v1,
  gen52v2doubles,
  gen5anythinggoes,
  gen5balancedhackmons,
  gen5customgame,
  gen5doublescustomgame,
  gen5doublesou,
  gen5gbudoubles,
  gen5gbusingles,
  gen5lc,
  gen5monotype,
  gen5nu,
  gen5ou,
  gen5oublind,
  gen5pu,
  gen5ru,
  gen5smogondoubles,
  gen5ubers,
  gen5uu,
  gen5vgc2013,
  gen5zu,
  gen61v1,
  gen6almostanyability,
  gen6anythinggoes,
  gen6balancedhackmons,
  gen6battlefactory,
  gen6battlespotdoubles,
  gen6battlespotsingles,
  gen6battlespottriples,
  gen6cap,
  gen6customgame,
  gen6doublesou,
  gen6doublesuu,
  gen6lc,
  gen6mixandmega,
  gen6monotype,
  gen6nu,
  gen6ou,
  gen6ounoteampreview,
  gen6pu,
  gen6purehackmons,
  gen6randombattle,
  gen6ru,
  gen6triplescustomgame,
  gen6ubers,
  gen6uu,
  gen6vgc2016,
  gen71v1,
  gen71v1suspecttest,
  gen72017noholdsbarred,
  gen72v2doubles,
  gen7350cup,
  gen7almostanyability,
  gen7almostanyabilitysuspecttest,
  gen7alolafriendly,
  gen7alphabetcup,
  gen7anythinggoes,
  gen7anythinggoesbeta,
  gen7averagemons,
  gen7balancedhackmons,
  gen7balancedhackmonssuspecttest,
  gen7battlefactory,
  gen7battleofalola,
  gen7battlespotdoubles,
  gen7battlespotsingles,
  gen7battlespotspecial1,
  gen7battlespotspecial10,
  gen7battlespotspecial11,
  gen7battlespotspecial12,
  gen7battlespotspecial13,
  gen7battlespotspecial14,
  gen7battlespotspecial15,
  gen7battlespotspecial16,
  gen7battlespotspecial17,
  gen7battlespotspecial2,
  gen7battlespotspecial3,
  gen7battlespotspecial4,
  gen7battlespotspecial5,
  gen7battlespotspecial6,
  gen7battlespotspecial7,
  gen7battlespotspecial8,
  gen7battlespotspecial9,
  gen7benjaminbutterfree,
  gen7bssfactory,
  gen7camomons,
  gen7cap,
  gen7caplc,
  gen7challengecup,
  gen7challengecup1v1,
  gen7challengecup2v2,
  gen7challengecupbeta,
  gen7chimera,
  gen7crossevolution,
  gen7customgame,
  gen7doublesanythinggoesbeta,
  gen7doublescustomgame,
  gen7doubleshackmonscup,
  gen7doublesou,
  gen7doublesoubeta,
  gen7doublesoususpecttest,
  gen7doublesubers,
  gen7doublesuu,
  gen7dragoncup,
  gen7dualwielding,
  gen7followtheleader,
  gen7fortemons,
  gen7fullpotential,
  gen7godlygift,
  gen7godsandfollowers,
  gen7hackmonscup,
  gen7hiddentype,
  gen7inheritance,
  gen7inverse,
  gen7johtoxalola,
  gen7kantoxalolaregionalrumble,
  gen7lastwill,
  gen7lc,
  gen7lcsuspecttest,
  gen7letsgo1v1,
  gen7letsgocup,
  gen7letsgodoublesnorestrictions,
  gen7letsgodoublesou,
  gen7letsgohackmons,
  gen7letsgoou,
  gen7letsgosinglesnorestrictions,
  gen7letsgouu,
  gen7linked,
  gen7mediocremons,
  gen7megamelee,
  gen7megamons,
  gen7mergemons,
  gen7metagamiate,
  gen7metronomebattle,
  gen7middlecup,
  gen7mixandmega,
  gen7mixandmegalc,
  gen7mixandmegasuspecttest,
  gen7monotype,
  gen7monotypesuspecttest,
  gen7natureswap,
  gen7nfe,
  gen7nu,
  gen7nualpha,
  gen7nubeta,
  gen7nususpecttest,
  gen7ou,
  gen7oubeta,
  gen7oublitz,
  gen7ouprebank,
  gen7oususpecttest,
  gen7partnersincrime,
  gen7petmodcleanslatemicro,
  gen7pokbilities,
  gen7pokebankag,
  gen7pokebankanythinggoes,
  gen7pokebankanythinggoesbeta,
  gen7pokebankdoublesag,
  gen7pokebankdoublesou,
  gen7pokebankdoublesoubeta,
  gen7pokebankdoublesubers,
  gen7pokebanklc,
  gen7pokebankou,
  gen7pokebankoubeta,
  gen7pokebankoususpecttest,
  gen7pokebankubers,
  gen7pokebankuualpha,
  gen7pokebilities,
  gen7prebankou,
  gen7pu,
  gen7pualpha,
  gen7pubeta,
  gen7purehackmons,
  gen7pususpecttest,
  gen7randombattle,
  gen7randomdoublesbattle,
  gen7reversed,
  gen7ru,
  gen7rualpha,
  gen7rubeta,
  gen7rususpecttest,
  gen7scalemons,
  gen7sharedpower,
  gen7simplesymphony,
  gen7sketchmons,
  gen7sketchmonssuspecttest,
  gen7stabmons,
  gen7stabmonssuspecttest,
  gen7suicidecup,
  gen7tiershift,
  gen7tinytourney,
  gen7trademarked,
  gen7ubers,
  gen7uberssuspecttest,
  gen7ultimatez,
  gen7ultrafinal,
  gen7ultrakalosclassic,
  gen7ultranoholdsbarred,
  gen7ultrasinnohclassic,
  gen7ultraspookycup,
  gen7uu,
  gen7uubeta,
  gen7uususpecttest,
  gen7vgc2017,
  gen7vgc2017beta,
  gen7vgc2018,
  gen7vgc2019moonseries,
  gen7vgc2019sunseries,
  gen7vgc2019ultraseries,
  gen7weaknesscup,
  gen7zu,
  gen81v1,
  gen82v2doubles,
  gen8350cup,
  gen8aaadoubles,
  gen8almostanyability,
  gen8alphabetcup,
  gen8alternatium,
  gen8anythinggoes,
  gen8anythinggoesobtainable,
  gen8anythinggoeswitheternamax,
  gen8armorbeginnings,
  gen8balancedhackmons,
  gen8balancedhackmonsbeta,
  gen8battleoflegends,
  gen8battlestadiumdoubles,
  gen8battlestadiumdoublesseries13,
  gen8battlestadiumsingles,
  gen8bdsp3v3singles,
  gen8bdspbalancedhackmons,
  gen8bdspbattlefestivaldoubles,
  gen8bdspdoublesou,
  gen8bdsplc,
  gen8bdspmonotype,
  gen8bdspnu,
  gen8bdspou,
  gen8bdsppu,
  gen8bdspru,
  gen8bdspubers,
  gen8bdspuu,
  gen8bonustype,
  gen8brokenrecord,
  gen8camomons,
  gen8cap,
  gen8cap1v1,
  gen8categoryswap,
  gen8championleague,
  gen8chimera1v1,
  gen8crossevolution,
  gen8doublescustomgame,
  gen8doublesou,
  gen8doublesoubeta,
  gen8doublesoudynamaxlevel0,
  gen8doublesubers,
  gen8doublesuu,
  gen8dragonkingcup,
  gen8extremespeedsinglebattles,
  gen8fairandsquare,
  gen8firstblood,
  gen8flipped,
  gen8fu,
  gen8fusionevolutionuu,
  gen8galarbeginnings,
  gen8galarnewcomers,
  gen8godlygift,
  gen8gymchallenge,
  gen8ichoosechu,
  gen8inheritance,
  gen8inverse,
  gen8jumpmagikarp,
  gen8lc,
  gen8lcsuspecttest,
  gen8linked,
  gen8maxberries,
  gen8megamax,
  gen8megasforall,
  gen8metronomebattle,
  gen8mixandmega,
  gen8monotype,
  gen8monotypedynamax,
  gen8multibility,
  gen8mythicalmelee,
  gen8natdexagwitheternamax,
  gen8natdexanythinggoes,
  gen8nationaldex,
  gen8nationaldexag,
  gen8nationaldexbeta,
  gen8nationaldexbh,
  gen8nationaldexmonotype,
  gen8nationaldexnostall,
  gen8nationaldexuu,
  gen8nationaldexuunostall,
  gen8natureswap,
  gen8ndmonotypenotoxapex,
  gen8nfe,
  gen8nu,
  gen8nususpecttest,
  gen8ou,
  gen8oubeta,
  gen8oublitz,
  gen8oususpecttest,
  gen8pacifistmons,
  gen8partnersincrime,
  gen8pokebilities,
  gen8pokebilitiesaaa,
  gen8pu,
  gen8puindeedeefnoteampreview,
  gen8purehackmons,
  gen8reevolution,
  gen8revelationmons,
  gen8ru,
  gen8rudynamax,
  gen8samedoublebeat,
  gen8scalemons,
  gen8sharedpower,
  gen8singlebattlecup,
  gen8sketchmons,
  gen8spikemuthcup,
  gen8spookycup2021,
  gen8staaabmons,
  gen8stabmons,
  gen8stabmonsmixandmega,
  gen8tagteamsingles,
  gen8testyourluck,
  gen8thelosersgame,
  gen8tiershift,
  gen8tiershiftcamomonsobtainable,
  gen8tinytourney,
  gen8trademarked,
  gen8tundratourney,
  gen8ubers,
  gen8ultimatefinale,
  gen8uu,
  gen8uususpecttest,
  gen8vgc2020,
  gen8vgc2021,
  gen8vgc2021series10,
  gen8vgc2021series11,
  gen8vgc2021series9,
  gen8vgc2021series9limitonerestrictedrestrictedlegendary,
  gen8vgc2022,
  gen8viabilities,
  gen8waterparadise,
  gen8zu,
  gen91v1,
  gen92v2doubles,
  gen9350cup,
  gen9almostanyability,
  gen9alphabetcup,
  gen9anythinggoes,
  gen9balancedhackmons,
  gen9battlestadiumdoubles,
  gen9battlestadiumsingles,
  gen9battlestadiumsinglesregulationc,
  gen9battlestadiumsinglesregulationd,
  gen9battlestadiumsinglesregulatione,
  gen9battlestadiumsinglesseries1,
  gen9battlestadiumsinglesseries2,
  gen9bonustype,
  gen9brokencup,
  gen9camomons,
  gen9cap,
  gen9convergence,
  gen9crossevolution,
  gen9doubleslc,
  gen9doublesou,
  gen9doublesubers,
  gen9doublesuu,
  gen9forceofthefallen,
  gen9fortemons,
  gen9franticfusions,
  gen9freedomcup,
  gen9fullpotential,
  gen9godlygift,
  gen9inheritance,
  gen9kitakamiprologue,
  gen9lc,
  gen9meowcup,
  gen9metronomebattle,
  gen9mixandmega,
  gen9mixandmegacup,
  gen9monotype,
  gen9nationaldex,
  gen9nationaldexag,
  gen9nationaldexbh,
  gen9nationaldexdoubles,
  gen9nationaldexmonotype,
  gen9nationaldexubers,
  gen9nationaldexuu,
  gen9nfe,
  gen9nu,
  gen9ou,
  gen9oususpect,
  gen9paldeaprologue,
  gen9partnersincrime,
  gen9partnersincrimesuspect,
  gen9pokebilities,
  gen9pu,
  gen9ru,
  gen9sharedpower,
  gen9sharingiscaring,
  gen9stabmons,
  gen9teradonation,
  gen9thecardgame,
  gen9tiershift,
  gen9trademarked,
  gen9trickmagic,
  gen9ubers,
  gen9uu,
  gen9vgc2023regulationc,
  gen9vgc2023regulationd,
  gen9vgc2023regulatione,
  gen9vgc2023regulationebo3,
  gen9vgc2023series1,
  gen9vgc2023series2,
  gen9voltturnmayhem,
  gen9zu,
  generationshowdown,
  giftsofthegods,
  godsandfollowers,
  gottalent,
  hackmons1v1,
  hackmonscup,
  hiddentype,
  higheststatmeta,
  inheritance,
  inversebattle,
  johtoclassic,
  kantoclassic,
  lcextended,
  lcsuspecttest,
  lcuu,
  linked,
  megamons,
  metagamiate,
  monogen,
  monsjustmons,
  mrboneswildride,
  natureswap,
  noguardgalaxy,
  nostatus,
  nususpecttest,
  ounomega,
  oususpecttest,
  pgllittlecup,
  pikachucup,
  primalbattle,
  proteanpalace,
  pususpecttest,
  randomdoublesbattle,
  randomtriplesbattle,
  returnd,
  rususpecttest,
  sametypestealthrock,
  scrappyskirmish,
  seasonaldimensiondoom,
  seasonalhanvshun,
  seasonalpolaropposites,
  seasonalrainbowroad,
  seasonalsleighshowdown,
  seasonalspacetimefuntimes,
  seasonalspoopyparty,
  seasonalstrikesback,
  seasonalsupersquadsmackdown,
  seasonalsuperstaffbros,
  seasonalyouarenotprepared,
  sinnohclassic,
  sketchmons,
  smogondoubles,
  smogondoublessuspecttest,
  smogondoublesubers,
  smogondoublesuu,
  smogontriples,
  spookycup,
  stabmons,
  statswitch,
  tiershift,
  typereflector,
  uberssuspecttest,
  unovaclassic,
  unreleasedou,
  uunoscald,
  uususpecttest,
  vgc2014,
  vgc2015,
  voltturnmayhem
};
export {
  aaaubers,
  abilityunity,
  averagemons,
  battlecup1v1,
  battleofhoenn,
  battlespotspecial10,
  battlespotspecial11,
  battlespotspecial12,
  battlespotspecial13,
  battlespotspecial17,
  battlespotspecial7,
  battlespotspecial8,
  bhdoubles,
  capcrucibelleplaytest,
  capnaviathanplaytest,
  capplasmantaplaytest,
  challengecup,
  challengecup1v1,
  challengecup1vs1,
  classichackmons,
  custompower,
  latest_default as default,
  doublesoususpecttest,
  doublesubers,
  dragoncup,
  enchanteditems,
  enterthedragontype,
  extremetiershift,
  festivefeud,
  finaldestination,
  flashclash,
  followtheleader,
  fu,
  gen11v1,
  gen1challengecup,
  gen1customgame,
  gen1nintendocup1997,
  gen1nobig4ou,
  gen1nu,
  gen1ou,
  gen1oubeta,
  gen1oumew,
  gen1ounodefenseboosting,
  gen1ounodefensecurlhardenorreflect,
  gen1ounotauros,
  gen1pu,
  gen1randombattle,
  gen1stabmons,
  gen1stadium,
  gen1stadiumou,
  gen1tradebacksou,
  gen1ubers,
  gen1uu,
  gen21v1,
  gen2inverseou,
  gen2mediocremons,
  gen2nc2000crystalmoves,
  gen2nu,
  gen2ou,
  gen2oubeta,
  gen2oublitz,
  gen2ouhoohcelebi,
  gen2ouitemclause,
  gen2ounosnorlax,
  gen2pu,
  gen2stabmons,
  gen2ubers,
  gen2uu,
  gen2uubl,
  gen2uumuk,
  gen31v1,
  gen3adv200,
  gen3bl,
  gen3customgame,
  gen3doublesou,
  gen3hoenngaiden,
  gen3lc,
  gen3nu,
  gen3ou,
  gen3oubeta,
  gen3oulatias,
  gen3ounospikes,
  gen3ounotyranitar,
  gen3stabmons,
  gen3tiershift,
  gen3ubers,
  gen3ubersbeta,
  gen3uu,
  gen3uubl,
  gen41v1,
  gen4anythinggoes,
  gen4balancedhackmons,
  gen4customgame,
  gen4doublesou,
  gen4lc,
  gen4monotype,
  gen4nu,
  gen4ou,
  gen4oulatimence,
  gen4outeampreview,
  gen4pu,
  gen4ubers,
  gen4uu,
  gen4zu,
  gen51v1,
  gen52v2doubles,
  gen5anythinggoes,
  gen5balancedhackmons,
  gen5customgame,
  gen5doublescustomgame,
  gen5doublesou,
  gen5gbudoubles,
  gen5gbusingles,
  gen5lc,
  gen5monotype,
  gen5nu,
  gen5ou,
  gen5oublind,
  gen5pu,
  gen5ru,
  gen5smogondoubles,
  gen5ubers,
  gen5uu,
  gen5vgc2013,
  gen5zu,
  gen61v1,
  gen6almostanyability,
  gen6anythinggoes,
  gen6balancedhackmons,
  gen6battlefactory,
  gen6battlespotdoubles,
  gen6battlespotsingles,
  gen6battlespottriples,
  gen6cap,
  gen6customgame,
  gen6doublesou,
  gen6doublesuu,
  gen6lc,
  gen6mixandmega,
  gen6monotype,
  gen6nu,
  gen6ou,
  gen6ounoteampreview,
  gen6pu,
  gen6purehackmons,
  gen6randombattle,
  gen6ru,
  gen6triplescustomgame,
  gen6ubers,
  gen6uu,
  gen6vgc2016,
  gen71v1,
  gen71v1suspecttest,
  gen72017noholdsbarred,
  gen72v2doubles,
  gen7350cup,
  gen7almostanyability,
  gen7almostanyabilitysuspecttest,
  gen7alolafriendly,
  gen7alphabetcup,
  gen7anythinggoes,
  gen7anythinggoesbeta,
  gen7averagemons,
  gen7balancedhackmons,
  gen7balancedhackmonssuspecttest,
  gen7battlefactory,
  gen7battleofalola,
  gen7battlespotdoubles,
  gen7battlespotsingles,
  gen7battlespotspecial1,
  gen7battlespotspecial10,
  gen7battlespotspecial11,
  gen7battlespotspecial12,
  gen7battlespotspecial13,
  gen7battlespotspecial14,
  gen7battlespotspecial15,
  gen7battlespotspecial16,
  gen7battlespotspecial17,
  gen7battlespotspecial2,
  gen7battlespotspecial3,
  gen7battlespotspecial4,
  gen7battlespotspecial5,
  gen7battlespotspecial6,
  gen7battlespotspecial7,
  gen7battlespotspecial8,
  gen7battlespotspecial9,
  gen7benjaminbutterfree,
  gen7bssfactory,
  gen7camomons,
  gen7cap,
  gen7caplc,
  gen7challengecup,
  gen7challengecup1v1,
  gen7challengecup2v2,
  gen7challengecupbeta,
  gen7chimera,
  gen7crossevolution,
  gen7customgame,
  gen7doublesanythinggoesbeta,
  gen7doublescustomgame,
  gen7doubleshackmonscup,
  gen7doublesou,
  gen7doublesoubeta,
  gen7doublesoususpecttest,
  gen7doublesubers,
  gen7doublesuu,
  gen7dragoncup,
  gen7dualwielding,
  gen7followtheleader,
  gen7fortemons,
  gen7fullpotential,
  gen7godlygift,
  gen7godsandfollowers,
  gen7hackmonscup,
  gen7hiddentype,
  gen7inheritance,
  gen7inverse,
  gen7johtoxalola,
  gen7kantoxalolaregionalrumble,
  gen7lastwill,
  gen7lc,
  gen7lcsuspecttest,
  gen7letsgo1v1,
  gen7letsgocup,
  gen7letsgodoublesnorestrictions,
  gen7letsgodoublesou,
  gen7letsgohackmons,
  gen7letsgoou,
  gen7letsgosinglesnorestrictions,
  gen7letsgouu,
  gen7linked,
  gen7mediocremons,
  gen7megamelee,
  gen7megamons,
  gen7mergemons,
  gen7metagamiate,
  gen7metronomebattle,
  gen7middlecup,
  gen7mixandmega,
  gen7mixandmegalc,
  gen7mixandmegasuspecttest,
  gen7monotype,
  gen7monotypesuspecttest,
  gen7natureswap,
  gen7nfe,
  gen7nu,
  gen7nualpha,
  gen7nubeta,
  gen7nususpecttest,
  gen7ou,
  gen7oubeta,
  gen7oublitz,
  gen7ouprebank,
  gen7oususpecttest,
  gen7partnersincrime,
  gen7petmodcleanslatemicro,
  gen7pokbilities,
  gen7pokebankag,
  gen7pokebankanythinggoes,
  gen7pokebankanythinggoesbeta,
  gen7pokebankdoublesag,
  gen7pokebankdoublesou,
  gen7pokebankdoublesoubeta,
  gen7pokebankdoublesubers,
  gen7pokebanklc,
  gen7pokebankou,
  gen7pokebankoubeta,
  gen7pokebankoususpecttest,
  gen7pokebankubers,
  gen7pokebankuualpha,
  gen7pokebilities,
  gen7prebankou,
  gen7pu,
  gen7pualpha,
  gen7pubeta,
  gen7purehackmons,
  gen7pususpecttest,
  gen7randombattle,
  gen7randomdoublesbattle,
  gen7reversed,
  gen7ru,
  gen7rualpha,
  gen7rubeta,
  gen7rususpecttest,
  gen7scalemons,
  gen7sharedpower,
  gen7simplesymphony,
  gen7sketchmons,
  gen7sketchmonssuspecttest,
  gen7stabmons,
  gen7stabmonssuspecttest,
  gen7suicidecup,
  gen7tiershift,
  gen7tinytourney,
  gen7trademarked,
  gen7ubers,
  gen7uberssuspecttest,
  gen7ultimatez,
  gen7ultrafinal,
  gen7ultrakalosclassic,
  gen7ultranoholdsbarred,
  gen7ultrasinnohclassic,
  gen7ultraspookycup,
  gen7uu,
  gen7uubeta,
  gen7uususpecttest,
  gen7vgc2017,
  gen7vgc2017beta,
  gen7vgc2018,
  gen7vgc2019moonseries,
  gen7vgc2019sunseries,
  gen7vgc2019ultraseries,
  gen7weaknesscup,
  gen7zu,
  gen81v1,
  gen82v2doubles,
  gen8350cup,
  gen8aaadoubles,
  gen8almostanyability,
  gen8alphabetcup,
  gen8alternatium,
  gen8anythinggoes,
  gen8anythinggoesobtainable,
  gen8anythinggoeswitheternamax,
  gen8armorbeginnings,
  gen8balancedhackmons,
  gen8balancedhackmonsbeta,
  gen8battleoflegends,
  gen8battlestadiumdoubles,
  gen8battlestadiumdoublesseries13,
  gen8battlestadiumsingles,
  gen8bdsp3v3singles,
  gen8bdspbalancedhackmons,
  gen8bdspbattlefestivaldoubles,
  gen8bdspdoublesou,
  gen8bdsplc,
  gen8bdspmonotype,
  gen8bdspnu,
  gen8bdspou,
  gen8bdsppu,
  gen8bdspru,
  gen8bdspubers,
  gen8bdspuu,
  gen8bonustype,
  gen8brokenrecord,
  gen8camomons,
  gen8cap,
  gen8cap1v1,
  gen8categoryswap,
  gen8championleague,
  gen8chimera1v1,
  gen8crossevolution,
  gen8doublescustomgame,
  gen8doublesou,
  gen8doublesoubeta,
  gen8doublesoudynamaxlevel0,
  gen8doublesubers,
  gen8doublesuu,
  gen8dragonkingcup,
  gen8extremespeedsinglebattles,
  gen8fairandsquare,
  gen8firstblood,
  gen8flipped,
  gen8fu,
  gen8fusionevolutionuu,
  gen8galarbeginnings,
  gen8galarnewcomers,
  gen8godlygift,
  gen8gymchallenge,
  gen8ichoosechu,
  gen8inheritance,
  gen8inverse,
  gen8jumpmagikarp,
  gen8lc,
  gen8lcsuspecttest,
  gen8linked,
  gen8maxberries,
  gen8megamax,
  gen8megasforall,
  gen8metronomebattle,
  gen8mixandmega,
  gen8monotype,
  gen8monotypedynamax,
  gen8multibility,
  gen8mythicalmelee,
  gen8natdexagwitheternamax,
  gen8natdexanythinggoes,
  gen8nationaldex,
  gen8nationaldexag,
  gen8nationaldexbeta,
  gen8nationaldexbh,
  gen8nationaldexmonotype,
  gen8nationaldexnostall,
  gen8nationaldexuu,
  gen8nationaldexuunostall,
  gen8natureswap,
  gen8ndmonotypenotoxapex,
  gen8nfe,
  gen8nu,
  gen8nususpecttest,
  gen8ou,
  gen8oubeta,
  gen8oublitz,
  gen8oususpecttest,
  gen8pacifistmons,
  gen8partnersincrime,
  gen8pokebilities,
  gen8pokebilitiesaaa,
  gen8pu,
  gen8puindeedeefnoteampreview,
  gen8purehackmons,
  gen8reevolution,
  gen8revelationmons,
  gen8ru,
  gen8rudynamax,
  gen8samedoublebeat,
  gen8scalemons,
  gen8sharedpower,
  gen8singlebattlecup,
  gen8sketchmons,
  gen8spikemuthcup,
  gen8spookycup2021,
  gen8staaabmons,
  gen8stabmons,
  gen8stabmonsmixandmega,
  gen8tagteamsingles,
  gen8testyourluck,
  gen8thelosersgame,
  gen8tiershift,
  gen8tiershiftcamomonsobtainable,
  gen8tinytourney,
  gen8trademarked,
  gen8tundratourney,
  gen8ubers,
  gen8ultimatefinale,
  gen8uu,
  gen8uususpecttest,
  gen8vgc2020,
  gen8vgc2021,
  gen8vgc2021series10,
  gen8vgc2021series11,
  gen8vgc2021series9,
  gen8vgc2021series9limitonerestrictedrestrictedlegendary,
  gen8vgc2022,
  gen8viabilities,
  gen8waterparadise,
  gen8zu,
  gen91v1,
  gen92v2doubles,
  gen9350cup,
  gen9almostanyability,
  gen9alphabetcup,
  gen9anythinggoes,
  gen9balancedhackmons,
  gen9battlestadiumdoubles,
  gen9battlestadiumsingles,
  gen9battlestadiumsinglesregulationc,
  gen9battlestadiumsinglesregulationd,
  gen9battlestadiumsinglesregulatione,
  gen9battlestadiumsinglesseries1,
  gen9battlestadiumsinglesseries2,
  gen9bonustype,
  gen9brokencup,
  gen9camomons,
  gen9cap,
  gen9convergence,
  gen9crossevolution,
  gen9doubleslc,
  gen9doublesou,
  gen9doublesubers,
  gen9doublesuu,
  gen9forceofthefallen,
  gen9fortemons,
  gen9franticfusions,
  gen9freedomcup,
  gen9fullpotential,
  gen9godlygift,
  gen9inheritance,
  gen9kitakamiprologue,
  gen9lc,
  gen9meowcup,
  gen9metronomebattle,
  gen9mixandmega,
  gen9mixandmegacup,
  gen9monotype,
  gen9nationaldex,
  gen9nationaldexag,
  gen9nationaldexbh,
  gen9nationaldexdoubles,
  gen9nationaldexmonotype,
  gen9nationaldexubers,
  gen9nationaldexuu,
  gen9nfe,
  gen9nu,
  gen9ou,
  gen9oususpect,
  gen9paldeaprologue,
  gen9partnersincrime,
  gen9partnersincrimesuspect,
  gen9pokebilities,
  gen9pu,
  gen9ru,
  gen9sharedpower,
  gen9sharingiscaring,
  gen9stabmons,
  gen9teradonation,
  gen9thecardgame,
  gen9tiershift,
  gen9trademarked,
  gen9trickmagic,
  gen9ubers,
  gen9uu,
  gen9vgc2023regulationc,
  gen9vgc2023regulationd,
  gen9vgc2023regulatione,
  gen9vgc2023regulationebo3,
  gen9vgc2023series1,
  gen9vgc2023series2,
  gen9voltturnmayhem,
  gen9zu,
  generationshowdown,
  giftsofthegods,
  godsandfollowers,
  gottalent,
  hackmons1v1,
  hackmonscup,
  hiddentype,
  higheststatmeta,
  inheritance,
  inversebattle,
  johtoclassic,
  kantoclassic,
  lcextended,
  lcsuspecttest,
  lcuu,
  linked,
  megamons,
  metagamiate,
  monogen,
  monsjustmons,
  mrboneswildride,
  natureswap,
  noguardgalaxy,
  nostatus,
  nususpecttest,
  ounomega,
  oususpecttest,
  pgllittlecup,
  pikachucup,
  primalbattle,
  proteanpalace,
  pususpecttest,
  randomdoublesbattle,
  randomtriplesbattle,
  returnd,
  rususpecttest,
  sametypestealthrock,
  scrappyskirmish,
  seasonaldimensiondoom,
  seasonalhanvshun,
  seasonalpolaropposites,
  seasonalrainbowroad,
  seasonalsleighshowdown,
  seasonalspacetimefuntimes,
  seasonalspoopyparty,
  seasonalstrikesback,
  seasonalsupersquadsmackdown,
  seasonalsuperstaffbros,
  seasonalyouarenotprepared,
  sinnohclassic,
  sketchmons,
  smogondoubles,
  smogondoublessuspecttest,
  smogondoublesubers,
  smogondoublesuu,
  smogontriples,
  spookycup,
  stabmons,
  statswitch,
  tiershift,
  typereflector,
  uberssuspecttest,
  unovaclassic,
  unreleasedou,
  uunoscald,
  uususpecttest,
  vgc2014,
  vgc2015,
  voltturnmayhem
};
